### ISSUE TITLE
1817 better wording when too many tokens

### ISSUE THREAD

**Issue by ChrisTheCat** [tobymao/18xx#2655]
In the image below, there are two sets of instructions to the player. In the black bar above the log, it tells you to "Choose tokens to remove to drop below 8." In the log, which is closer to the map where the action must be taken, it tells you to "decide which tokens to keep."

The actual action that must be taken is selecting which tokens *not* to keep. Suggest rewording the log to say "decide which tokens to remove" to match the text in the black bar above.

<img width="619" alt="Screen Shot 2020-12-15 at 10 45 54 AM" src="https://user-images.githubusercontent.com/8732435/102258373-ba977e00-3ec2-11eb-8121-e4637b56702c.png">

**Comment by jenf** [comment]
fixed
